core/templates/dev/head/components/forms/schema_editors/schema_based_html_editor_directive.html

```diff
@@ -1,6 +1,6 @@
-<text-angular-rte ng-if="!isDisabled()" html-content="$parent.localValue"
-                  ui-config="uiConfig()" label-for-focus-target="labelForFocusTarget()">
-</text-angular-rte>
+<ck-editor-rte ng-if="!isDisabled()" ng-model="$parent.localValue"
```

Comment:
Until the migration is ready, let's use a boolean flag in `constants.js` which controls whether to show text-angular or ck-editor-rte. By default, we should still be showing textAngular.

Reply:
Done!